From: Michael Biebl <biebl@debian.org>
Date: Sun, 23 Jul 2017 18:56:22 +0200
Subject: Remove leftover @LIBTRACKER_MINER_PC_REQUIRES@ from
 tracker-miner.pc.in

Follow-up for commit ac84484dfb5dea636e4b2e830d25901413fb970c.

https://bugzilla.gnome.org/show_bug.cgi?id=785294
---
 src/libtracker-miner/tracker-miner.pc.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libtracker-miner/tracker-miner.pc.in b/src/libtracker-miner/tracker-miner.pc.in
index 386ba72..36a8a32 100644
--- a/src/libtracker-miner/tracker-miner.pc.in
+++ b/src/libtracker-miner/tracker-miner.pc.in
@@ -7,6 +7,5 @@ Name: tracker-miner
 Description: A library to develop tracker data miners
 Version: @VERSION@
 Requires: glib-2.0 gio-2.0 tracker-sparql-@TRACKER_API_VERSION@
-Requires.priv: @LIBTRACKER_MINER_PC_REQUIRES@
 Libs: -L${libdir} -ltracker-miner-@TRACKER_API_VERSION@
 Cflags: -I${includedir}/tracker-@TRACKER_API_VERSION@
